src/runtime.js

```diff
@@ -513,6 +513,36 @@ export class MultidocManager {
       amp.resources = Services.resourcesForDoc(ampdoc);
     }
 
+    /**
+     * Expose amp-bind methods
+     */
+    amp['bind'] = {
+      /**
+       * Expose amp-bind getState
+       * @param {string} name - Name of state or deep state
+       * @return {Promise<(?JsonObject|string|undefined)>} - Resolves to
+       * a copy of the value of a state
+       */
+      getState: name => {
+        return Services.bindForDocOrNull(shadowRoot).then(bind => {
+          return bind && bind.getStateCopy(name);
+        });
+      },
+
+      /**
+       * Expose amp-bind setState
+       * @param {!JsonObject} state - State to be set
+       * @return {Promise} - Resolves after state and history have been updated
+       */
+      setState: state => {
+        return Services.bindForDocOrNull(shadowRoot).then(bind => {
+          return (
+            (bind && bind.setStateAndUpdateHistory(state)) || Promise.resolve()
+          );
```

Comment:
This would be more readable as:
`return bind ? bind.setStateAndUpdateHistory(state) : Promise.resolve();`